src/validator.js

```diff
@@ -471,4 +474,21 @@ export default class Validator {
       return ARCH_DEFAULT;
     }
   }
+
+  _markEmptyFiles(addonMetadata) {
+    var emptyFiles = [];
+
+    return this.io.getFiles()
+      .then((files) => {
+        for (let filename in files) {
+          if (files[filename].uncompressedSize === 0) {
```

Comment:
Actually, scrap that, seems cleaner to check for uncompressedSize then just size. I’ll have a patch in a few.